util/src/builder/callback_data.rs

```diff
@@ -0,0 +1,175 @@
+use twilight_model::{
+    application::{callback::CallbackData, component::Component},
+    channel::{
+        embed::Embed,
+        message::{AllowedMentions, MessageFlags},
+    },
+};
+
+/// Builder for [`CallbackData`].
+///
+/// # Example
+/// ```
+/// use twilight_util::builder::CallbackDataBuilder;
+/// use twilight_model::channel::message::MessageFlags;
+///
+/// let callback_data = CallbackDataBuilder::new()
+///     .content("My callback message")
+///     .flags(MessageFlags::EPHEMERAL)
+///     .build();
+/// ```
```

Comment:
Can we move this example to a module-level doc comment?

Reply:
Do you want to move it to the `builder` module documentation? As the `callback_data` module contains only one struct, I thought that re-exporting the type directly to the root of the `builder` and making `callback_data` module private was better.

Reply:
I mean in the same file, just having it at the top with `//!` comments. See `cache/in-memory/src/permission.rs` for an example

Reply:
Which appears like this: https://api.twilight.rs/twilight_cache_inmemory/permission/index.html

Reply:
Yes, but actually the `util::builder::callback_data` is not exported, only the `CallbackDataBuilder` type is re-exported in the `builder` module. Do you want to export the `callback_data` module entirely?

Reply:
I'll have to get back to you on this, I'll pull in the code and play around with how it renders, then let you know.